receiver/redisreceiver/metadata.yaml

```diff
@@ -15,14 +20,14 @@ attributes:
 
 metrics:
   redis.maxmemory:
-    enabled: false
+    enabled: true
```

Comment:
Why changing them to enabled by default? I don't remember we agreed on this, and I still don't think they are essential metrics

Reply:
I thought adding resource attribute is already a breaking change, so why not change them to true by default as well.

Reply:
Because I don't believe that these metrics are important enough to be emitted by default

Reply:
I changed them back